book/src/configuration.md

```diff
@@ -27,8 +27,8 @@ hidden = false
 
 You can use a custom configuration file by specifying it with the `-c` or
 `--config` command line argument, for example `hx -c path/to/custom-config.toml`.
-Additionally, you can reload the configuration file by sending the USR1
-signal to the Helix process on Unix operating systems, such as by using the command `pkill -USR1 hx`.
+You can "hot-reload" the config file by issuing the `:config-reload` command. Alternatively, on Unix operating systems, you can reload it by sending the USR1
```

Comment:
```suggestion
You can reload the config file by issuing the `:config-reload` command. Alternatively, on Unix operating systems, you can reload it by sending the USR1
```

I think it's clearer to just say "reload"

Reply:
Sounds good.

Reply:
Actually, you're also more correct, since it is not a "hot reload" if user takes explicit action (since technically hot reload implies watching for file changes)